virtual_library_card/settings/dev.py

```diff
@@ -22,6 +22,7 @@
         "PASSWORD": "test",
         "HOST": os.environ.get("VLC_DEV_DB_HOST", "pg"),
         "PORT": os.environ.get("VLC_DEV_DB_PORT", "5432"),
+        "OPTIONS": {"sslmode": os.environ.get("PG_SSL_MODE", "require")},
```

Comment:
I think this should follow the same naming scheme we already have for these environment variables. Based on the name, I assumed it was a PG environment variable and I was looking in the PG docs to try to figure out what it did, until I found it defined here.

I'd suggest something like `VLC_DEV_DB_SSL_MODE`.